Source/WebCore/css/typedom/CSSStyleValue.h

```diff
@@ -110,7 +110,11 @@ class CSSStyleValue : public RefCounted<CSSStyleValue>, public ScriptWrappable {
 public:
     String toString() const;
     virtual void serialize(StringBuilder&, OptionSet<SerializationArguments> = { }) const;
-    virtual ~CSSStyleValue() = default;
+    virtual ~CSSStyleValue()
```

Comment:
I'd rather we silence bad GCC warning some other way. Defining a non-default destructor can have side effects we don't want.

Reply:
I can try IGNORE_WARNINGS instead, sure.

That said, I'm not sure what the side effects would be. This changes order of destruction, but it shouldn't matter.

Reply:
OK, IGNORE_WARNINGS works fine here. It's less pretty this way IMO, but reviewer gets to choose. ;)